### PR TITLE
docs: sd-link docs [skip chromatic]

### PR DESCRIPTION
**Pull request by smfonseca**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/components/link/link.stories.ts
Comment (mariohamann) [inline packages/components/src/components/link/link.stories.ts:1]: Can we add default `href`, `target` and `download` as done here as well?
https://github.com/solid-design-system/solid/pull/1226/files#r1701323337

File: packages/components/src/components/link/link.stories.ts [hunk 1]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" standalone>
+        <sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>
+        Standalone
+      </sd-link>
+
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/">
+        <sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>
+        Without Standalone
+      </sd-link>
+    </div>
```
[the remark this answers is not in the document]
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts]: Done

File: packages/components/src/components/link/link.stories.ts [hunk 2]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
-      args,
-      options: {
-        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      }
-    })
-};
-
-/**
- * Use the `icon-left` and `icon-right` slots to add icons. They automatically adapt the size.
+ * Use the `size` attribute to adjust the size of the link. By default it is set to `inherit` but can be changed to `lg` or `sm`.
```
[the remark this answers is not in the document]
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts]: Done

File: packages/components/src/components/link/link.stories.ts
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts:1]: Done

File: packages/components/src/components/link/link.test.stories.ts [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+      },
+      args,
+      options: {
+        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
+      }
+    })
+};
+
+/**
+ * Use the `icon-left` and `icon-right` slots to add icons. They automatically adapt the size.
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.test.stories.ts:94]: * Use the `icon-left` and `icon-right` slots to add system icons. They automatically adapt the size.

File: packages/components/src/components/link/link.stories.ts [hunk 4]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
+    <div class="flex gap-12">
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/">Enabled</sd-link>
+      <sd-link>Disabled</sd-link>
+    </div>
+  `
 };
 
 /**
- * Use the `base`, `label`, `icon-left` and `icon-right` part selectors to customize the button.
+ * Use the `icon-left` and `icon-right` slots to add icons to each side of the link. They automatically adapt the size.
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:138]: * Use the `icon-left` and `icon-right` slots to add system icons to each side of the link. They automatically adapt the size.

File: packages/components/src/components/link/link.stories.ts [hunk 5]
```diff
@@ -2,19 +2,24 @@
...
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes } = storybookDefaults('sd-link');
 const { generateTemplate } = storybookTemplate('sd-link');
 const { overrideArgs } = storybookHelpers('sd-link');
 
+/**
+ *
+ * This component is used to display links and can also be used to perform actions like downloading files.
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:13]: Allows users to navigate to another location or perform actions like e.g. downloading files.

File: packages/components/src/components/link/link.stories.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (coraliefeil) [reply packages/components/src/components/link/link.stories.ts]: Add blank line between link list and text.

File: packages/components/src/components/link/link.stories.ts [hunk 6]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
-      args
-    });
-  }
+export const SlottedIcons = {
+  name: 'Slotted Icons',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" standalone>
+        <sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>
+        Slotted Icon Left
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:147]: Icon Left

File: packages/components/src/components/link/link.stories.ts [hunk 7]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
+      </sd-link>
+
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" standalone>
+        Slotted Icon Right
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:151]: Icon Right

File: packages/components/src/components/link/link.stories.ts [hunk 8]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
-          type: 'slot',
-          name: 'icon-left',
-          value: '<sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>'
-        }
-      ],
-      args
-    });
-  }
+export const SlottedIcons = {
+  name: 'Slotted Icons',
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:142]: Icon

**Comment by coraliefeil** [comment]
See commments.
Also refined design docs -> Pls see [figma docs](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2009-2177&t=DabI7KfPr1jcqQjB-11)

**Review by mariohamann — APPROVED** [review]

File: packages/components/src/components/link/link.stories.ts (hunk 4 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts:140]: Updated

File: packages/components/src/components/link/link.stories.ts [hunk 9]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
+      <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" inverted>Inverted</sd-link>
+    </div>
+  `
 };
 
 /**
- * This `standalone` attribute controls the layout of the icon and text within the component.
- * - If true, the icon and text will be displayed side by side, each occupying its own column.
- * - If false or not provided, the icon will be displayed inline within the text.
+ * Use the `standalone` attribute to control the layout of the icon and text within the component.
```
[the remark this answers is not in the document]
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts:70]: Updated

File: packages/components/src/components/link/link.stories.ts (hunk 6 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts:149]: Updated

File: packages/components/src/components/link/link.stories.ts (hunk 7 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts:153]: Updated

File: packages/components/src/components/link/link.stories.ts (hunk 8 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts:144]: Updated

File: packages/components/src/components/link/link.stories.ts (hunk 5 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts]: Updated

File: packages/components/src/components/link/link.stories.ts [hunk 10]
```diff
@@ -32,297 +37,131 @@ export const Default = {
...
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-link');
-    await waitUntil(() => el?.shadowRoot?.querySelector('a'));
-    // We have to catch the event as otherwise Storybook will break
-    el!.addEventListener('click', e => e.preventDefault());
-    await userEvent.type(el!.shadowRoot!.querySelector('a')!, '{return}', { pointerEventsCheck: 0 });
-  }
+export const Sample = {
```
[the remark this answers is not in the document]
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts]: Moved

File: packages/components/src/components/link/link.stories.ts [hunk 11]
```diff
@@ -32,297 +39,119 @@ export const Default = {
...
+          <li>
+            <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" standalone>
+              <sd-icon library="global-resources" name="system/pen" slot="icon-left"></sd-icon>
+              Solid Design System
+            </sd-link>
+          </li>
+        </ul>
+
+        <p>
+          While the list above shows <code>standalone</code> links, we now will link to the
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:68]: While the list above shows standalone links, we now will link to the

remove <code> pls, the font does not show the markup correctly.

File: packages/components/src/components/link/link.stories.ts [hunk 12]
```diff
@@ -32,297 +39,119 @@ export const Default = {
...
+          <sd-link href="https://cd.union-investment.de">CD Toolbox</sd-link> inside a paragraph.
+        </p>
+      </div>
+    </div>
+  `
 };
 
 /**
- * Icons automatically adapt to the link's invertedness.
+ * Use the `size` attribute to adjust the size of the link. By default it is set to `inherit` to adapt to the surrounding text but can be explicitly changed to `lg` or `sm`.
```
Comment (coraliefeil) [inline packages/components/src/components/link/link.stories.ts:77]: * Use the `size` attribute to adjust the size of the link. By default it is set to `inherit` to adapt to the surrounding text but can be changed to `lg` or `sm`.

**Comment by coraliefeil** [comment]
2 more comments then it is good.

File: packages/components/src/components/link/link.stories.ts (hunk 12 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts]: Updated

File: packages/components/src/components/link/link.stories.ts (hunk 11 above)
Comment (smfonseca) [reply packages/components/src/components/link/link.stories.ts]: Updated

**Comment by coraliefeil** [comment]
Fine with me. THX

**Review by mariohamann — APPROVED** [review]
Let's ship it! @smfonseca

**Pull request merged**